frontend/dockerfile/dockerfile_mount_test.go

```diff
@@ -115,7 +115,7 @@ RUN --mont=target=/mytmp,type=tmpfs /bin/true
 		},
 	}, nil)
 	require.Error(t, err)
-	require.Contains(t, err.Error(), "unknown flag: mont")
+	require.Contains(t, err.Error(), "unknown flag: --mont")
 	require.Contains(t, err.Error(), "did you mean mount?")
```

Comment:
We should probably also look at this one to say `did you mean --mount?`, but I think the same error-type is used in multiple situations, so it probably needs some changes to make it preserve `--`, but that not to affect the "matching"; https://github.com/moby/buildkit/blob/185751bdc010f1f8bf1192d6e9b4ec19552d5741/util/suggest/error.go#L58-L70


Maybe the `--` could be trimmed as part of the Search (not sure), but then it would have to be added again when printing, so this needs a slightly more deeper dive into how this all works;
https://github.com/moby/buildkit/blob/185751bdc010f1f8bf1192d6e9b4ec19552d5741/util/suggest/error.go#L47